src/main/java/step2/domain/Lotto.java

```diff
@@ -55,15 +28,20 @@ public ScoreType addWinningInfos(WinnersNo winnersNo) {
         return hitNumber > 2 ? ScoreType.getScore(hitNumber, hasBonusNumber) : ScoreType.NONE;
```

Comment:
저번에 코멘트 남겨드렸던것 같은데.. 반영이 안되어 있네요.

```suggestion
        return ScoreType.getScore(hitNumber, hasBonusNumber);
```

getScore에서 `.orElse(NONE);`를 해주기 때문에 불필요한 코드들로 보입니다.

Reply:
네넵 맞습니다. 저번에 남긴 코멘트에서 null을 -> ScoreType.NONE 변경하였고
이번에 filer에서 NONE을 제외 하는 코드를 사용하면서 하신 말씀의 의미를 알았습니다!